config/routes.yaml

```diff
@@ -0,0 +1,3 @@
+kernel:
+    resource: ../context/Kernel.php
+    type: annotation
```

Comment:
I'd remove this routes config, I don't expect anyone using the kernel class as controller on this skeleton